project/ProjectPlugin.scala

```diff
@@ -158,37 +151,21 @@ object ProjectPlugin extends AutoPlugin {
 
   }
 
-  import autoImport._
-
   override def projectSettings: Seq[Def.Setting[_]] =
-    Seq(
-      resolvers += Resolver.bintrayRepo("beyondthelines", "maven"),
-      orgAfterCISuccessTaskListSetting := List(
-        depUpdateDependencyIssues.asRunnableItem,
-        orgPublishReleaseTask
-          .asRunnableItem(allModules = true, aggregated = false, crossScalaVersions = true),
-        orgUpdateDocFiles.asRunnableItem
-      ),
+    scalaMetaSettings ++ sharedReleaseProcess ++ warnUnusedImport ++ Seq(
+      libraryDependencies ++= commonDeps :+ %("slf4j-nop") % Test,
+      Test / fork := true,
+      Tut / scalacOptions := (Compile / console / scalacOptions).value,
+      orgAfterCISuccessTaskListSetting ~= (_.filterNot(_ == defaultPublishMicrosite)),
```

Comment:
Replaces previous fully-specified `orgAfterCISuccessTaskListSetting` with a diff from the imported default.